packages/core/src/util/pg.ts

```diff
@@ -0,0 +1,28 @@
+import { Pool } from "pg";
+import { DbCreds, dbCredsSchema, dbReadReplicaEndpointSchema } from "./sequelize";
+// Initialize the main DB pool
+export function initDbPool(dbCreds: DbCreds) {
+  const parsedDbCreds = dbCredsSchema.parse(dbCreds);
+  const pool = new Pool({
+    user: parsedDbCreds.username,
+    host: parsedDbCreds.host,
+    database: parsedDbCreds.dbname,
+    password: parsedDbCreds.password,
+    port: parsedDbCreds.port,
+  });
```

Comment:
It would be good to have a const and set the pool size (property `max`). Even though the default 10 connections is fine for this use case, we're building something that might be used afterwards, so being explicit about this, with a [link to the docs](https://node-postgres.com/apis/pool), is helpful - I had to search it up online to make sure there was more than 1 connection by default.